docker/Dockerfile.pdo-dev

```diff
@@ -48,8 +48,8 @@
 #   - for develooping based on source in host you might map source into container with an option
 #     like -v $(pwd):/project/pdo/src/private-data-objects/
 
```

Comment:
This pdo-dev docker file still fetches `ccf-0.11.7` below. Is it needed?

Reply:
fixed.